### DISCUSSION TITLE
Allow to confirm the partecipation to a live

### DISCUSSION THREAD

**Discussion by albx** [albx/KITT#88] [Ideas]
Add a link to the notification message to confirm if I will partecipate to the scheduled livestream.

This could be a link to an empty page which only register the partecipation, or it could redirect the user to a simple form where confirm the presence.

Any other ideas?!

**Comment by totti240282** [comment]
I would open a page where there is a thank you message: "Thank you for registering for the event" or something similar, to let the customer know that something has happened.
